src/App.js

```diff
@@ -1,13 +1,144 @@
-import react from 'react';
+import axios from 'axios';
+import { useEffect, useState, useRef } from 'react';
 import './App.css';
+import Footer from './components/Footer';
+import Header from './components/Header.js';
+import News from './components/News.js';
+import Bitcoin from './components/Bitcoin.js';
+import CryptoNews from "./components/CryptoNews.js";
+
 
 function App() {
+
+  const [bitcoin, setBit] = useState([]);
+  const [userInput, setUserInput] = useState('');
+  const [searchBit, setSearchBit] = useState('');
+
+//CRIPTOCURRENCY API
+
+  const firstUpdate = useRef(true);
+
+  useEffect(() => {
+    if (firstUpdate.current) {
+      firstUpdate.current = false;
+      return;
```

Comment:
This is the first time I'm seeing a return without specifying what is being returned, just out of curiousity, how did this work?